.github/workflows/gh-pages_test.yml

```diff
@@ -4,21 +4,17 @@ on:
   push:
     branches:
       - master
-      - documentation
+      - gh-pages
```

Comment:
ott minek fut le?

Reply:
@hegyhati az én eredeti ötletem az `on: pull-request` volt `on: \ push: \ branches: \` helyett, azonban valamelyik mergeléssel mégis ez a verzió került be, egy másik branchről. 

Amiatt gondoltam, hogy akkor deployoljon, amikor PR van, mert minden pushra túl sok folyamat és teletolja a history-t, merge addig meg úgy sem megy be, ameddig nincs meg minden check, illetve ha PR alatt commit van, akkor is újrafut. A masteren a push-t azért tartom butaságnak, mert merge indítja el a pusht masterre, magyarul ha a deploy elszáll, akkor nem lehet módosítani. Javítom egy hotfixben mindjárt

Reply:
igazabol itt a logikus az, hogy a masterre megy a deplot. a feature brancheken felesleges a deploy-nak lefutnia, ez azt jelenti, hogy push sorrendtol fuggetlenul (PR or not) tobb kulonbozo valtozat valtogatna egymast. Annak lenne ertelme, hogy lenne egy "szemet"  deploy branch, ahova mindenki aki nem a master pushol, hogyha ellenorizni akarod, hogy lemegy-e szepen. 

de a gh-pages-re semmi ertelme beletenni ezt, mert ott a fajlok kozott nincs is ott ez a yaml file, ott csak a dokumentacio van (remelem), tehat ott nem fog semmi lefutni.